lib/src/main/java/uk/gov/di/ipv/core/library/persistence/item/AccessTokenItem.java

```diff
@@ -11,6 +11,7 @@ public class AccessTokenItem implements DynamodbItem {
     private String ipvSessionId;
     private String revokedAtDateTime;
     private long ttl;
+    private String expiryDateTime;
```

Comment:
Not a big deal, but I called this `accessTokenExpiryDateTime` in passport-back just to avoid confusing it with the `ttl` which is the expiry of the dynamodb record